example/spec/s3Access/s3AccessSpec.js

```diff
@@ -82,32 +81,32 @@ async function canListObjects(region, credentials) {
   return invokeTestLambda(region, credentials, 'list-objects');
 }
 
-let server;
-
 process.env.AccessTokensTable = `${config.stackName}-AccessTokensTable`;
 const accessTokensModel = new AccessToken();
 
 describe('When accessing an S3 bucket directly', () => {
   let accessToken;
 
-  beforeAll(async (done) => {
+  beforeAll(async () => {
     await s3().putObject({ Bucket: protectedBucket, Key: testFileKey, Body: 'test' }).promise();
     setDistributionApiEnvVars();
```

Comment:
Do we still need this...?

Reply:
Yes, to get a token from EDL you need to have the redirect available and in your EDL application